docs/gemstones/rpaste.md

```diff
@@ -0,0 +1,79 @@
+---
+title: rpaste - Pastebin Tool
+author: Steven Spencer
+contributors:
+tags:
+  - rpaste
+  - Mattermost
+  - pastebin
+---
+
+# Introduction to `rpaste`
+
```

Comment:
1. In the third sentence, can we reference the subject again? "This tool is particularly useful when..." 
2. Still in that sentence let's streamline an adverb and remove ""particularly"
3. And then in the next sentence can we change "extremely" to "especially"? It's not wrong, it's just nuance.
4. Next sentence, let's reference the subject again: "The 'rpaste' tool can be installed..." or 'rpaste' can be installed..."

Reply:
not sure how this got attributed to line 12 instead of 13.